### ISSUE TITLE
Bug: No detail in ReductError exception

### ISSUE THREAD

**Issue by atimin** [reduct-storage/reduct-py#50]
### What happened?

Reduct Storage returns additional information if it executed a HTTP request with an error:

```
{
 "detail": "Some message"}
}
```

However, the library doesn't show it. It has only status

### Version

1.0.0

### OS

Linux

### Snippet to repeat the bug

```python
from reduct import Client, ReducError
from asyncio import new_event_loop

client = Client("https://play.reduct-storage.dev", apit_token="reduct")

try:
  new_event_loop().run_until_complete(cl.get_bucket("DOES-NOT-EXIST"))
catch RuductError as err:
  print(err)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by aschenbecherwespe** [comment]
Problem is that HEAD shouldn't ever be returning a response body which is normally used for the error messages, I'll update for a fix...

**Comment by atimin** [comment]
Yep, I'll fix this problem on the server side in the v1.1 release. See reduct-storage/reduct-storage#197

**Comment by aschenbecherwespe** [comment]
I think it's fine just to use GET though, right? Since we want an error message. I've made that change anyway...
